pyproject.toml

```diff
@@ -5,21 +5,34 @@ authors = [
 ]
 description = "Python + Aegisub"
 name = "Pyegi"
-version = "0.0.1"
+version = "0.1.0"
 
 [tool.poetry.dependencies]
-Pillow = "^9.0.1"
-PyQt6 = "^6.2.3"
-pyonfx = "^0.9.13"
-python = ">=3.9,<3.10"
-pyqtdarktheme = "^1.1.0"
-toml = "^0.10.2"
+Pillow = "9.*"
+PyQt6 = "6.*"
```

Comment:
Just a reminder:
Since there is a warning in PyQt6 v6.5 which hasn't been resolved yet, please modify this line as follows:
PyQt6 = "6.4.*"
or:
PyQt6 = "6.*,<6.5"
Also, please add this line:
PyQt6-Qt6 = "6.4.*"
or:
PyQt6-Qt6 = "6.*,<6.5"

Reply:
حالا در هر صورت گذاشتمش روی همینی که گفتی.

Reply:
دوباره بگیر پایجی رو و تست کن. توی آپدیت جدید ۶.۵.۲ شاید این مشکلی که گفته بودی رفع شده باشه. نتیجه رو خبر بده.

Reply:
پایجی رو کلاً از نو دانلود کردم و نصب کردم. نصب شد. توی اجی هم بالا میاد ولی توی کامند پرامپت دوباره همین رو می‌نویسه:
qt.qpa.window: SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2) failed: COM error 0x5: Access is denied.
Qt's default DPI awareness context is DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2. If you know what you are doing you can overwrite this default using qt.conf (https://doc.qt.io/qt-6/highdpi.html#configuring-windows)